docs/Release.md

```diff
@@ -0,0 +1,38 @@
+#Release
+In adding updating function to addressbook, its release is tweaked to separate the dependencies libraries (their JARS)
+from the main app, so that only affected JARs need to be updated. This saves bandwidth in downloading update.
```

Comment:
Explain how things are now (and why), rather than how you arrived there. This phrase sound like you are describing how you arrived there `In adding updating function to addressbook, its release is tweaked`